test/nodetool/test_cluster.py

```diff
@@ -0,0 +1,375 @@
+#
+# Copyright 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+
+import pytest
+
+from test.nodetool.utils import check_nodetool_fails_with_error_contains, check_nodetool_fails_with
+from test.nodetool.rest_api_mock import expected_request
+
+
+def _remove_log_timestamp(res):
+    """ Log timestamp[1] is impossible to match accurately, so just remove it
+
+    E.g. for:
+
+        [2023-12-22 09:18:06,615] Repair session 1
+
+    We drop the [2023-12-22 09:18:06,615] part
+
+    Also strip trailing the whitespace, that nodetool loves to add randomly.
```

Comment:
This new nodetool command doesn't have a Java nodetool equivalent, so no random trailing whitespaces should be added anymore :).